codec-http/src/test/java/io/netty/handler/codec/http/websocketx/extensions/compression/PerMessageDeflateDecoderTest.java

```diff
@@ -86,21 +90,20 @@ public void testNormalFrame() {
         // test
         assertNotNull(newFrame);
         assertNotNull(newFrame.content());
-        assertTrue(newFrame instanceof BinaryWebSocketFrame);
```

Comment:
Why was this removed ?

Reply:
https://github.com/netty/netty/blob/721d2c02da639bcc61cd1659921591018a4c7f3d/codec-http/src/test/java/io/netty/handler/codec/http/websocketx/extensions/compression/PerMessageDeflateDecoderTest.java#L88